specification/compute/resource-manager/Microsoft.Compute/stable/2021-07-01/compute.json

```diff
@@ -9610,10 +9613,30 @@
             "name": "VirtualMachineSizeTypes",
             "modelAsString": true
           }
+        },
+        "vmSizeProperties": {
+          "$ref": "#/definitions/VMSizeProperties",
+          "description": "Specifies the VM Size Properties for customizing VM sizes."
```

Comment:
Specifies the properties for customizing VM sizes.

Reply:
ack

Reply:
Sameer had also suggested to add min- api version in the description text.. is that still needed?